blackbox-exporter.json

```diff
@@ -1,8 +1,8 @@
 {
   "__inputs": [
     {
-      "name": "DS_PROMETHEUS",
-      "label": "Prometheus",
+      "name": "DS_LOCALHOST",
+      "label": "localhost",
```

Comment:
i'm not sure this is right.

this is what i got in cache-health.json for example:

```
  "__inputs": [
    {
      "name": "DS_PROMETHEUS",
      "label": "Prometheus",
      "description": "",
      "type": "datasource",
      "pluginId": "prometheus",
      "pluginName": "Prometheus"
    }
  ],
```

... and it loads correctly when the dashboard gets "provisionned" (ie. reloaded by grafana after a puppet run).

how did you export this dashboard?

Reply:
the dashboard in grafana is complaining https://grafana2.torproject.org/d/NgEq8C0Zz/blackbox-exporter?orgId=1 abot DS_PROMETHEUS not being found and in your fullnode exporter you use DS_LOCALHOST ...

Reply:
https://github.com/hiromipaw/grafana-dashboards/blob/master/node-exporter-full.json

Reply:
honestly I didn't export the dashboard.. just restarted grafana...

Reply:
> in your fullnode exporter you use DS_LOCALHOST

gah. oops. well i guess the original prometheus node is kind of a mess and has *multiple* data sources like this. i wouldn't take that one as canon - i mess about quite a bit with those dashboards.

leave the default that was provided in the original dashboard you imported, and creating a matching datasource in grafana. it doesn't matter if there are many.